controllers/fenceagentsremediation_controller_test.go

```diff
@@ -273,47 +416,28 @@ func createVA(vaName, unhealthyNodeName string) *storagev1.VolumeAttachment {
 // cleanupTestedResources fetches all the resources that we have crated for the test
 // and if they are still exist at the end of the test, then we clean them up for next test
 func cleanupTestedResources(va1, va2 *storagev1.VolumeAttachment, pod *corev1.Pod) {
-	// clean test volume attachments if it exists
-	vaTest := &storagev1.VolumeAttachment{}
-	if err := k8sClient.Get(context.Background(), client.ObjectKeyFromObject(va1), vaTest); err == nil {
-		log.Info("Cleanup: clean volume attachment", "va name", vaTest.Name)
-		Expect(k8sClient.Delete(context.Background(), vaTest)).To(Succeed())
+	for _, va := range []*storagev1.VolumeAttachment{va1, va2} {
+		vaTest := &storagev1.VolumeAttachment{}
+		key := client.ObjectKeyFromObject(va)
+		if err := k8sClient.Get(context.Background(), key, vaTest); err == nil {
+			log.Info("Cleanup: clean volume attachment", "va name", vaTest.Name)
+			Expect(k8sClient.Delete(context.Background(), vaTest)).To(Succeed())
+		}
 	}
-	if err := k8sClient.Get(context.Background(), client.ObjectKeyFromObject(va2), vaTest); err == nil {
-		log.Info("Cleanup: clean volume attachment", "va name", vaTest.Name)
-		Expect(k8sClient.Delete(context.Background(), vaTest)).To(Succeed())
 
-	}
-	// clean test pod if it exists
 	podTest := &corev1.Pod{}
-	if err := k8sClient.Get(context.Background(), client.ObjectKeyFromObject(pod), podTest); err == nil {
+	key := client.ObjectKeyFromObject(pod)
+	if err := k8sClient.Get(context.Background(), key, podTest); err == nil {
 		log.Info("Cleanup: clean pod", "pod name", podTest.Name)
-		Expect(k8sClient.Delete(context.Background(), podTest, forced)).To(Succeed())
-	}
-}
-
-// isEqualStringLists return true if two string lists share the same values
-func isEqualStringLists(s1, s2 []string) bool {
-	sort.Strings(s1)
-	sort.Strings(s2)
-	return reflect.DeepEqual(s1, s2)
-}
 
-// cliCommandsEquality creates the command for CLI and compares it with the production command
-func cliCommandsEquality(far *v1alpha1.FenceAgentsRemediation) (bool, error) {
-	if mocksExecuter.command == nil {
-		return false, errors.New("The command from mocksExecuter is null")
+		// Delete the resource immediately
```

Comment:
I don't think the next lines are needed.
`Expect(k8sClient.Delete(context.Background(), podTest, forced)).To(Succeed())` is good enough IMHO.

Reply:
Correct, I rewrote this. However, I prefer to keep the `force` variable locally